### DISCUSSION TITLE
When will version 5.0.0 be released ? Any date ?

### DISCUSSION THREAD

**Discussion by dibakarbose** [TomSchimansky/CustomTkinter#572] [General]
When will version 5.0.0 be released ? Any date ?

**Comment by TomSchimansky** [comment]
Maybe tomorrow night if I don't find any bugs...

**Comment by Hancie123** [comment]
It is still not available? When version 5 will be available? Any fix date?

**Comment by deepak4u** [comment]
Loved your work🔥 I'm eagerly waiting for Version 5.0 to work on CTkTabview.

**Comment by dibakarbose** [comment]
@TomSchimansky : Any date for ver 5.0.0 release ?  Eagerly waiting for it. 😊

**Comment by TomSchimansky** [comment]
Version 5.0.0 is released now!
